package.json

```diff
@@ -49,11 +49,7 @@
     "webpack": "^5.89.0",
     "webpack-merge": "^5.9.0",
     "webpack-sources": "^3.2.3",
-    "webpack-stats-plugin": "^1.1.3",
-    "dotcom-tool-kit": "^3.3.7",
```

Comment:
*Question*: what is the reason for the toolkit package to be removed?

Reply:
https://github.com/Financial-Times/n-syndication/blob/main/package.json#L36-L53

Not being removed, but it (alongside some other packages) were duplicated in the package json, which is a non breaking error.

Reply:
Thanks for the cleanup on the duplicated packages 👍